test/service/test_response.py

```diff
@@ -2095,24 +2102,68 @@ def test_summary_filter_none(self):
 
 class TestUnpopulatedIndexResponse(WebServiceTestCase):
 
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return []
+
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.index_service.create_indices(cls.catalog)
+        cls._setup_indices()
+        cls.maxDiff = None
 
     @classmethod
     def tearDownClass(cls):
-        cls.index_service.delete_indices(cls.catalog)
+        cls._teardown_indices()
         super().tearDownClass()
 
+    def facets(self) -> Sequence[str]:
+        return self.app_module.app.service_config.facets
+
+    def entity_types(self) -> List[str]:
+        return [
+            entity_type for entity_type in self.index_service.entity_types(self.catalog)
+            if entity_type != 'cell_suspensions'
+        ]
+
     def test_empty_response(self):
-        url = self.base_url + "/index/projects"
-        response = requests.get(url)
-        response.raise_for_status()
-        response = response.json()
-        self.assertEqual([], response['hits'])
-        self.assertEqual({None}, set(response['pagination'].values()))
-        self.assertEqual({}, response['termFacets'])
+        for entity_type in self.entity_types():
+            with self.subTest(entity_type=entity_type):
+                response = requests.get(url=furl(url=self.base_url,
```

Comment:
```suggestion
                response = requests.get(url=furl(url=self.base_url,
```

I'd extract the furl object into a variable.

Reply:
My reasoning for keeping this inline with the method was that its only being used once during the get request, is there a reason to extract this out? Easy to read? 
Same with https://github.com/DataBiosphere/azul/pull/2396#discussion_r556011127

Reply:
Just for clarity.